src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/OrderPaymentDetailsModal/index.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Alert,
+  Button,
+  Modal,
+  ModalProps,
+  ModalSize,
+  VariantType,
+} from '@openfun/cunningham-react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { useState } from 'react';
+import { PaymentScheduleGrid } from 'components/PaymentScheduleGrid';
+import { CreditCard } from 'types/Joanie';
+import { CreditCardSelector } from 'components/CreditCardSelector';
+
+const messages = defineMessages({
+  title: {
+    id: 'components.DashboardItemOrder.PaymentModal.title',
+    defaultMessage: 'Payment details',
+    description: 'Title of the payment modal',
+  },
+  scheduleTitle: {
+    id: 'components.DashboardItemOrder.PaymentModal.scheduleTitle',
+    defaultMessage: 'Schedule',
```

Comment:
The right term looks to be "Repayment schedule"
```suggestion
    defaultMessage: 'Repayment schedule',
```